### ISSUE TITLE
1.0.6 mislabeled on pypi

### ISSUE THREAD

**Issue by leohemsted** [gaelen/python-statsd-client#5]
The filename of the 1.0.6 tar.gz on pypi is called `1.0.60` (presumably a typo), as seen here: https://pypi.org/project/statsd-client/1.0.6/#files.

This causes oddities and errors when trying to install - you can't pin to 1.0.6, and if you try and install 1.0.60 it ends up as 1.0.6 in pip.

(all the following in python 3.6.9, pip 19.2.1).

This breaks our build process (where we use `pip freeze` to create a pipfile-esque canonical requirements.txt) so we've had to pin to a git commit.

```
$ pip install statsd-client==1.0.6

Collecting statsd-client==1.0.6
  ERROR: Could not find a version that satisfies the requirement statsd-client==1.0.6 (from versions: 1.0.2, 1.0.4, 1.0.60)
ERROR: No matching distribution found for statsd-client==1.0.6

$ pip install statsd-client==1.0.60

Collecting statsd-client==1.0.60
Installing collected packages: statsd-client
Successfully installed statsd-client-1.0.6

$ pip freeze | grep statsd-client

statsd-client==1.0.6```

**Comment by Adam-Leyshon** [comment]
The new PIP Resolver refuses to install this package now because the filename doesn't match the metadata version.
We can't release a version without the Author of the package and we can't raise a PR since there's no code to fix.

Since this hasn't been updated for 5 years, do we consider this abandoned?
Should we fork this and fix it by creating yet-another-package?

**Comment by gaelen** [comment]
Sorry for the years of neglect. I'm not exactly an active developer anymore. I fixed this issue by upping the version to 1.0.7. I don't know what happened to the file that was 1.0.6 in PyPi. I must have messed something up in the original 1.0.6 file, and PiPy won't allow the same filename to be used, I lazily and called it 1.0.60. So lazy and definitely a little dumb. I didn't see a way around the filename issue without increasing the version, so here you have it - 1.0.7. Also, I apologize for committing straight to master. I thought I had a branch for it, but didn't till after I pushed, and followed through with my laziness by NOT popping the commit and doing it the right way. I was a better developer 9 years ago, but I like to think that I am a better person now.
